news/2021-05-24-results-monthly-beatmapping-contest-april-2020.md

```diff
@@ -0,0 +1,116 @@
+
+### Runner-up
+
+The **runner-up** this month is [uone](https://osu.ppy.sh/users/5321719), who executed a super pleasant wavy SV idea in his version of [*MYUKKE. - Ginevra*](https://www.dropbox.com/s/mt5tpb9kqo9vtsf/radar%20%28suffering%29%20-%20Creative%20Dolphin.osz?dl=0). The map includes patterns that play incredibly well, which should be no surprise considering his playing skill and mapping history. He deserves a big congratulations as well! 
+
+## osu!catch winners
+
+With a humble **14** entries, osu!catch skipped straight to the judging phase for all its entries!
+
+| Rank | Beatmap | Judging | Musical representation (50) | Creativity (50) | Gameplay (50) | Limitation (25) | Total (raw/standardized) |
+| :-- | :-- | :-- | :-- | :-- | :-- | :-- | :-- |
+| \#1 | [*Ginevra*](https://osu.ppy.sh/beatmapsets/1471164) by [autofanboy](https://osu.ppy.sh/users/636114) | [see details](https://mappersguild.com/contestresults?submission=6089e7e9f843d32ac49f0e68) | 48 | 40 | 42 | 21 | **151** (6.71) |
+| \#2 | [*OCCHOCO-REST-LESS*](https://osu.ppy.sh/beatmapsets/1471162) by [rew0825](https://osu.ppy.sh/users/2488026) | [see details](https://mappersguild.com/contestresults?submission=6089e7e7f843d32ac49f0e56) | 44 | 35 | 40 | 20 | **139** (4.09) |
+| \#3 | [*Ginevra*](https://osu.ppy.sh/beatmapsets/1471148) by [Deif](https://osu.ppy.sh/users/318565) | [see details](https://mappersguild.com/contestresults?submission=6089e7eaf843d32ac49f0e73) | 38 | 41 | 38 | 21 | **138** (3.59) |
+| \#4 | [*OCCHOCO-REST-LESS*](https://osu.ppy.sh/beatmapsets/1471157) by [-Ken](https://osu.ppy.sh/users/4430811) | [see details](https://mappersguild.com/contestresults?submission=6089e7e8f843d32ac49f0e63) | 38 | 35 | 34 | 21 | **128** (2.13) |
+| \#5 | [*Ginevra*](https://www.dropbox.com/s/q782zoupyduka6g/Monthly%20Beatmapping%20Contest%20April%202021%20%28osu%21catch%29%20%28Jumpy%20Pheasant%29.osz?dl=0) by [Sinnoh](https://osu.ppy.sh/users/4236057) | [see details](https://mappersguild.com/contestresults?submission=6089e7e9f843d32ac49f0e6d) | 32 | 39 | 31 | 21 | **123** (0.92) |
+
+[See results for all osu!catch submissions here](https://mappersguild.com/contestresults?contest=6089d5be1d4fc0755a574ab1)
+
+### Winner
+
+The long-awaited **winner** of the first iteration of osu!catch MBC is [autofanboy](https://osu.ppy.sh/users/636114)! Having already established his status as a celebrated veteran in the mapping community, autofanboy showcased his experience in his rendition of [*MYUKKE. - Ginevra*](https://osu.ppy.sh/beatmapsets/1471164), resulting in a beautifully-crafted beatmap that captivated the judges. Its intricate flow used this month's limitation more like a tool to explore ideas only viable in a no-slider map, rather than an obstacle that they had to map around. Make sure to give this wonderful map a try!
```

Comment:
```suggestion
The long-awaited **winner** of the first iteration of osu!catch MBC is [autofanboy](https://osu.ppy.sh/users/636114)! 

Having already established his status as a celebrated veteran in the mapping community, autofanboy showcased his experience in his rendition of [*MYUKKE. - Ginevra*](https://osu.ppy.sh/beatmapsets/1471164), resulting in a beautifully-crafted beatmap that captivated the judges. Its intricate flow used this month's limitation more like a tool to explore ideas only viable in a no-slider map, rather than an obstacle that they had to map around. Make sure to give this wonderful map a try!
```